### PR TITLE
Language tab

### PR DESCRIPTION
**Pull request by gobelius**
Added language tab for the plugin
Added WYSIWYG text editor with latest improvements

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -1,24 +1,32 @@
 {
-  "author":"BuildFire",
-  "pluginName":"Places 2.0",
-  "pluginDescription":"The Places 2.0 plugin is an amazing feature for showcasing multiple, categorized locations or points of interest. Your app users can view locations in a list or map view and filter based on the categories you set up and include pictures, contact information, instant directions, and even embed images and videos. Note: This plugin is not backwards compatible with Places 1.0 Keywords: restaurant, gps, navigation, franchise, pins. Keywords: Map, maps, geo",
-  "supportEmail":"support@buildfire.com",
-  "control":{
-    "content":{
-      "enabled":true
+  "author": "BuildFire",
+  "pluginName": "Places 2.0 Bego",
```
[the remark this answers is not in the document]
Comment (gobelius) [reply plugin.json]: Reverted, don't know how I missed it in the first place

File: src/widget/templates/detail.hbs [hunk 2]
```diff
@@ -77,18 +77,18 @@
...
                 <button id="contactBtn">
-                    Contact
+                    {{contactBtnText}}
                 </button>
             </div>
             {{else}}
             <div class="buttonItem long">
                 <button id="directionsBtn">
-                    Get Directions
+                    {{dirBtnText}}
```
Comment (mikailbayram) [inline src/widget/templates/detail.hbs:91]: Did we try to do this with bfString attribute? You can check out how its implemented in other plugins through this link: https://github.com/search?q=org%3ABuildFire+in%3Acode+bfstring&type=code

Comment (gobelius) [reply src/widget/templates/detail.hbs:91]: I did try it this way but couldn't make it work, so I got the string data in the details controller and passed them here

`dirBtnText: strings.get("ActionButtons.directionsButton").substring(0, 14) +  "..." || "Get Directions"`

File: plugin.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "author": "BuildFire",
-  "pluginName": "Places 2.0",
+  "pluginName": "Places 2.0 Bego",
```
Comment (mikailbayram) [inline plugin.json:3]: Please, please dont commit plugin.json file!

**Review by mikailbayram — CHANGES_REQUESTED** [review]

**Comment by mikailbayram** [comment]
Lets have the code formatted the way it is, I am constantly reviewing stuff that is just code formatting change @gobelius

**Review by mikailbayram — APPROVED** [review]

**Pull request merged**